src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CaseProceedsInCasemanRespondentNotificationHandler.java

```diff
@@ -55,7 +55,7 @@ private CallbackResponse notifyRespondentSolicitorForCaseProceedsInCaseman(Callb
 
         if (callbackParams.getRequest().getEventId() != null
             && callbackParams.getRequest().getEventId().equals("CREATE_CLAIM_SPEC")) {
-            if (stateFlowEngine.hasSpecTransitionedTo(callbackParams.getRequest().getCaseDetails(), CLAIM_NOTIFIED)) {
+            if (stateFlowEngine.hasTransitionedTo(callbackParams.getRequest().getCaseDetails(), CLAIM_NOTIFIED)) {
```

Comment:
This block of if code is exactly the same as the if block that follows it. Please remove the duplication

Reply:
The change made to remove the duplication seems to be incorrect and will perhaps affect existing functionality in the absence of the else condition. Please revisit this

Reply:
Change has been done to address this comment